src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  function getSum(firstNum) {
+    count++;
+
+    function add(secondNum) {
+      return count > 3 && count % 2 === 0
+        ? 'Bzzz... Error!'
+        : firstNum + secondNum;
```

Comment:
Try not to combine && and ternary operators.
```suggestion
      const shouldReturnError = count > 3 && count % 2;
      
      return shouldReturnError === 0
        ? 'Bzzz... Error!'
        : firstNum + secondNum;
```

Reply:
ooh, ok 👍